### PR TITLE
Sudipta/quickwindoc

### PR DESCRIPTION
**Pull request by khound**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,115 +48,38 @@
...
+The Photoshop APIs currently support creating and editing of Embedded Smart Objects. Support for Linked Smart Objects is forthcoming.
 
-We have not put a throttle limit on requests to the API at this time.
+- In order to update an embedded smart object that is referenced by multiple layers you only need to update one of those layers, the effect will be reflected in all layers referencing the same smart object.
 
-# Photoshop
+- The replaced smart object takes the bounds of the new image by default. If your document contains transparent pixels (e.g some .png) , you may not get consistent bounds.
 
-## General Workflow
+The API's are documented [here](https://adobedocs.github.io/photoshop-api-docs/#api-Photoshop-document_operations)
```
Comment (alshamma) [inline README.md]: I prefer the style of putting the link inside a title or phrase. E.g., 
[API Reference for documentOperations](https://adobedocs.github.io/photoshop-api-docs/#api-Photoshop-document_operations)

I think that reads better than having a dangling *here* as a link. 

Should this be a link for smartObject?

File: README.md [hunk 2]
```diff
@@ -60,115 +48,38 @@
...
 
-### Input and Output file storage
+[Smart Object Example Code](https://github.com/AdobeDocs/photoshop-api-docs#example-6-swapping-the-image-in-a-smart-object-layer)
 
-Clients can use assets stored on one of the following storage types:
-1. Adobe: by referencing the path to the files on Creative Cloud
-2. External: (like AWS S3) by using a presigned GET/PUT URL
-3. Azure: By generating a SAS (Shared Access Signature) for upload/download
-4. Dropbox: Generate temporary upload/download links using https://dropbox.github.io/dropbox-api-v2-explorer/
+Smart Object support is a work in progress.
```
Comment (alshamma) [inline README.md]: What do we mean by to convey to clients by the statement "is a work in progress"? 

I suggest instead we state what we know works, and what we know doesn't work.

File: README.md [hunk 3]
```diff
@@ -60,115 +48,38 @@
...
 
-### Text layers
+### Text layers (Supported in Pre-Release)
 
 The Photoshop APIs currently support creating and editing of Text Layer with different fonts, character styles and paragraph styles.
```
Comment (alshamma) [inline README.md]: Change to "text layers" instead of "Text Layer"

File: README.md [hunk 4]
```diff
@@ -190,36 +101,40 @@ If your font is not included in either of these locations you must include an hr
 
 Font support is a work in progress.
```
Comment (alshamma) [inline README.md]: Ditto. Instead state what works and what is not implemented. Don't make our readers guess;-)

**Comment by alshamma** [comment]
Are we doing a production release or a pre-release? If I am a developer writing for production, I think it would be confusing to see all the the pre-release stuff. I would urge that we have separate documents for production release and pre-release.

**Comment by noahhoffman** [comment]
@khound I built the api docs and  it doesn't look like you've separated out features correctly yet (maybe this is intentional and you're waiting for feedback?).  For example, the production section for docOps still shows blendOptions, fill, etc (and many more)

**Comment by khound** [comment]
@noahhoffman Definitely more fine tuning is needed. But `Prod` version of `/documentOperations` actually contains smartObject, fill, blendOptions, bounds and base layer functionalities like locked,visible,id and name.
Please refer this :
https://wiki.corp.adobe.com/display/DISysEng/APIs+for+PIE+SO+Workflow

**Comment by khound** [comment]
@noahhoffman When someone uses `/documentOperations` for `smartobjectlayer` and `filllayer`, `edit` and `add` need to be supported as well.

File: README.md (hunk 2 above)
Comment (khound) [reply README.md]: @alshamma Sure. Will make these changes once i gather all the review points.

**Pull request closed without merging**